Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -39,6 +40,14 @@ public class MenuItem : ItemsControl
         /// </summary>
         public static readonly DependencyProperty HeaderProperty = DependencyProperty.Register(nameof(Header), typeof(string), typeof(MenuItem), new PropertyMetadata(default(string)));
 
+        internal double X1 { get; private set; }
```

Comment:
Element can be moved at runtime. Probably best to just get the location of menu items within the menu itself while a menu is open